client/src/pages/Signup/SignupParents/SignupParents.tsx

```diff
@@ -299,6 +301,10 @@ const SignupParents = () => {
             </div>
           </FormGroup>
 
+          <Link to="/termsandconditions" className={styles.terms}>
```

Comment:
Also when i clcik signup there is no explanation about the different signups I can click